custom_components/tuya_local/devices/kcvents_vt501_fan.yaml

```diff
@@ -0,0 +1,153 @@
+name: Heat Recovery Ventilator
+products:
+  - id: 5p0togo3y3yzgp2p
+    name: KCvents VT-501
+primary_entity:
+  entity: fan
+  translation_only_key: fan_with_presets
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 101
+      type: boolean
+      name: speed
+      default: false
```

Comment:
I'm not sure what the intention of this is. `default` is not valid here.

Reply:
It was a left-over from playing around. Sorry, I forgot to re-check if really needed and didn't delete it.